docs/usage.md

```diff
@@ -1,178 +0,0 @@
----
-permalink: /usage/
-title: Usage
-lead: >
-  Install images, stylesheets, and script files in both their original and compiled forms with npm.
----
-
-```shell
-npm install identity-style-guide
-```
-
-If this is your first npm dependency, a `package-lock.json` file will be generated. Commit this file to source control.
-
-The module is now installed at `./node_modules/identity-style-guide`. Woohoo!
-
-## Precompiled CSS, JavaScript, fonts, and images
-
-You’ll find the compiled assets in `dist/assets`. These are ready to be used in your project directly.
-
-<div class="usa-alert usa-alert--warning usa-alert__paragraph">
-  <div class="usa-alert__body">
-    <p class="usa-alert__text" markdown="1">The paths in the compiled CSS file assume that your asset folders follow the same structure as those in `dist/assets`. Copy this folder in its entirety, or compile the assets from source as described below.</p>
-  </div>
-</div>
-
-We recommend including these files from `node_modules` directly, either by linking to them at that location or by copying them automatically during a build process, to make upgrades easier in the future.
-
-### Copy during Jekyll build process
-
-If you’re using Jekyll, a simple plugin can help copy this file during your build process to keep your assets up-to-date. First, add this file to `_plugins/`:
-
-```ruby
-# _plugins/copy_to_destination.rb
-
-module Jekyll
-  module CopyToDestination
-    class CopyGenerator < Generator
-      def generate(site)
-        folders = site.config['copy_to_destination'] || []
```

Comment:
can we add this jekyll plugin to the README? I've used it in the handbook

Reply:
Yep! Added here 61393d4